CHANGELOG.md

```
@@ -1,3 +1,15 @@
+## v0.20.9
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)